packages/components/src/navigator/stories/index.story.tsx

```diff
@@ -40,7 +40,7 @@ const meta: Meta< typeof NavigatorProvider > = {
 						 * detail of the Navigator component. Do not use outside of
 						 * its source code.
 						 */
-						[data-wp-component="NavigatorProvider"] {
+						[data-wp-component="Navigator"] {
 							height: 250px;
 						}
 						[data-wp-component="NavigatorScreen"]:not([data-sticky]) {
```

Comment:
I think we need to change this one too:

```suggestion
						[data-wp-component="Navigator.Screen"]:not([data-sticky]) {
```

Reply:
The current version is correct — we're setting the height on the wrapper, as also suggested in the README. Each screen, if the wrapper's height is set, will follow that height thanks to the grid layout.

Reply:
I think you're looking at the wrong place - this is about the padding, and in my testing the padding appears only if we use the dot notation.

Reply:
Alternatively, feel free to remove this style if you think we don't need it. But my point was that if you want the padding, you need to add the dot.

Reply:
You're right — I looked at the wrong diff 😅 

I pushed a fix as you suggested.